hw/xbox/nv2a/nv2a_psh.c

```diff
@@ -761,19 +879,32 @@ static QString* psh_convert(struct PixelShader *ps)
         if (ps->state.alpha_func == ALPHA_FUNC_NEVER) {
             qstring_append(ps->code, "discard;\n");
         } else {
-            const char* alpha_op;
+            const char *alpha_op;
             switch (ps->state.alpha_func) {
-            case ALPHA_FUNC_LESS: alpha_op = "<"; break;
-            case ALPHA_FUNC_EQUAL: alpha_op = "=="; break;
-            case ALPHA_FUNC_LEQUAL: alpha_op = "<="; break;
-            case ALPHA_FUNC_GREATER: alpha_op = ">"; break;
-            case ALPHA_FUNC_NOTEQUAL: alpha_op = "!="; break;
-            case ALPHA_FUNC_GEQUAL: alpha_op = ">="; break;
+            case ALPHA_FUNC_LESS:
+                alpha_op = "<";
+                break;
+            case ALPHA_FUNC_EQUAL:
+                alpha_op = "==";
+                break;
+            case ALPHA_FUNC_LEQUAL:
+                alpha_op = "<=";
+                break;
+            case ALPHA_FUNC_GREATER:
+                alpha_op = ">";
+                break;
+            case ALPHA_FUNC_NOTEQUAL:
+                alpha_op = "!=";
+                break;
+            case ALPHA_FUNC_GEQUAL:
+                alpha_op = ">=";
+                break;
```

Comment:
We should eventually move this into a seperate function probably.
I'm not sure if I like this longer version. I don't feel strongly about it, but I think the previous version with some alignment (table-style) would be the most readable, and avoid most accidental errors.